ci/Jenkinsfile.linux

```diff
@@ -21,7 +21,7 @@ pipeline {
   options {
     timestamps()
     /* Prevent Jenkins jobs from running forever */
-    timeout(time: 15, unit: 'MINUTES')
+    timeout(time: 20, unit: 'MINUTES')
```

Comment:
How are the timing changes related to the certificate update?

Reply:
they are not

Reply:
Gotcha! Just realized those changes live in https://github.com/status-im/status-desktop/pull/4144/commits/601f3e0d1a4b834794dc74cbe7f096a1f3c3c1e0